src/client/containers/BoardPageComponents/BoardAddCandidate/AddCandidate.js

```diff
@@ -1,7 +1,11 @@
-// import React, { useState } from 'react';
 import postData from './PostData';
 
-export default async function AddCandidate(newCandidate, userId, boardId) {
+export default async function AddCandidate(
+  newCandidate,
+  userId,
+  boardId,
+  setBoardLoading,
+) {
   try {
     if (!newCandidate.name) {
```

Comment:
We have some lint errors here also. Can you fix those?

Reply:
Are we going to show messages on the page in case of user don’t  put anything in in input or put numbers instead of strin?

Reply:
I have done that at first then we removed it.

Reply:
> Are we going to show messages on the page in case of user don’t put anything in the input or put numbers instead of string?

Lets add validation later. Would be nice if you could create an issue so we can remember